gulpfile.mjs

```diff
@@ -109,7 +159,10 @@ gulp.task('create-metadata', () =>
 
 gulp.task('create-iconmapping', createIconMapping);
 
-const allIcons = gulp.parallel('icons', 'copy-svgs');
+const allIcons = gulp.series(
+  gulp.parallel('optimise-small-icons', 'optimise-large-icons'),
+  gulp.parallel('icons', 'copy-svgs')
+);
```

Comment:
Here we are running the optimisation first to ensure the rest of the process uses the correct icons